scss/detail.scss

```diff
@@ -0,0 +1,85 @@
+.book-title {
+  height: 39px;
+  width: 210px;
+  color: #000;
```

Comment:
Now that you are using hex, we create a folder of variables  and here you can create a file _colors.scss where create varaibles for each color, after import this file and use on your styles,
For example :
`

_colors.scss
$black: #000;

styles.scss

@import 'variables/colors.scss'
 .book-title {
  color: $black;
}
`

Reply:
You can also use import without the extension.
`@import 'variables/colors'`